11-parallelism/src/lib.rs

```diff
@@ -0,0 +1,56 @@
+//! 11 Parallelism
+//! --------------
+//!
+//! Welcome to eleven step of this Rust workshop.
+//!
+//! This step focuses on basic multi threading API in Rust
+//!
+//! ## Thread
+//!
+//! To create a new thread, we call the `thread::spawn` function and pass it a **closure**.
```

Comment:
I would have added an example using a "function pointer"

Reply:
wha't the point with multi threading ?

Reply:
Because text specify closure it required to spawn a thread while clean code would suppose to use function pointer

Reply:
Done